src/Storage/Bucket.php

```diff
@@ -867,4 +1042,33 @@ private function isObjectNameRequired($data)
 
         return false;
     }
+
+    /**
+     * Return a topic name in its fully qualified format.
+     *
+     * @param Topic|string $topic
+     * @return string
+     * @throws \InvalidArgumentException
+     */
+    private function getFormattedTopic($topic)
+    {
+        if ($topic instanceof Topic) {
+            return sprintf('//pubsub.googleapis.com/%s', $topic->name());
```

Comment:
nit: you've got a constant with this value. could it be used as the first `sprintf` arg?

Reply:
Definitely!